src/petals/server/server.py

```diff
@@ -78,6 +78,8 @@ def __init__(
         load_in_8bit: Optional[bool] = None,
         tensor_parallel_devices: Optional[Sequence[torch.device]] = None,
         skip_reachability_check: bool = False,
+        use_relay: bool = True,
```

Comment:
It's best to either remove default argument values of to remove these arguments completely: we might forget to change defaults here in the future, and required values will be passed to kwargs anyway

Reply:
I see your point but disagree, since the convention in Petals is that all `Server` defaults match to the defaults of `run_server.py`.

Reply:
I will agree with @borzunov here. Here is my arguments:
1. use_relay will not be passed from run_server and we want it by default to be True
2. it is nice to see in the arguments all parameters that matters
3.  usually , it is not a good idea to be dependent on default argument from another library . They could be changed without notice and can lead to strange behavior

Reply:
I see your point about explicitly indicating arguments for creation of Server, though it somewhat contradicts the existence of **kwargs in init. My primary concern is that we should strive to have consistent defaults across different locations: one way to do this in an error-proof way would be to declare a common constant with the default value and use it in both locations. Besides, petals-cli is a part of Petals, so these files belong to the same library

Reply:
We would need to create constants for all defaults in this case (tens of them). I think this is a more general problem that should be addressed outside of this PR (maybe we should use smth like reflection).